dotCMS/src/main/java/com/dotcms/publisher/util/DependencyManager.java

```diff
@@ -476,14 +491,16 @@ private void setHostDependencies (final PublisherFilter publisherFilter) {
 
 				// Content dependencies
 				if(!publisherFilter.doesExcludeDependencyClassesContainsType(PusheableAsset.CONTENTLET.getType())) {
-					final String luceneQuery = "+conHost:" + host.getIdentifier();
-					final List<Contentlet> contentList = APILocator.getContentletAPI()
-							.search(luceneQuery, 0, 0, null, user, false);
-					for (final Contentlet contentlet : contentList) {
-						if(!publisherFilter.doesExcludeDependencyQueryContainsContentletId(contentlet.getIdentifier())) {
-							contents.addOrClean(contentlet.getIdentifier(),
-									contentlet.getModDate());
-							contentsSet.add(contentlet.getIdentifier());
+					if (!host.ifSystemHost()) {
```

Comment:
Codacy found an issue: [These nested if statements could be combined](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=6340082)